src/test/java/uk/gov/digital/ho/pttg/AuditEntryJpaRepositoryTest.java

```diff
@@ -181,11 +182,60 @@ public void findAuditHistory_filtersByEventType() {
         assertThat(all).size().isEqualTo(0);
     }
 
+    /**
+     * As this test involves some Postgres specific syntax for the JSONB column, the intention was to run this test
+     * against an in memory postgres database.  However, I was unable to get such a database working consistently in
+     * conjunction with Spring Boot.
+     *
+     * This test was run manually against postgres 9.6 running in a docker container, and it passed there, so we do at
+     * least know the test passes against a real postgres db.
+     *
+     * To run this test:
+     * - add the class level annotation @ActiveProfiles("postgres,postgresDocker")
+     * - run a docker container with the following command (replacing with the correct version of postgres):
+     *     docker run  --name pg-docker -e POSTGRES_PASSWORD=docker -d -p 5432:5432 postgres:9.6.12
+     * - remove the @Ignore annotation from the test
+     * You should now be able to run the test against the docker container.
+     */
+    @Ignore
```

Comment:
As you and Tim spent so much time on this you've gone for the right approach here. It would be nice if we could increase the likelihood that the next person to work in this area will remember to run the test to prevent regression. I'll think about it.

Perhaps a Gradle task that detects commits affecting the AuditEntryJpaRepository and does something like print a warning or fail the build until we 'force' it.

Reply:
It's there to prove that it works, not as a regression test.  Enough time has been spent proving that this works, I cannot justify any more.